Sasquatch/SasquatchObjC/Info.plist

```diff
@@ -35,6 +35,10 @@
 	<true/>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Select photo for attachment</string>
+	<key>NSLocationAlwaysUsageDescription</key>
```

Comment:
We don't need `NSLocationAlwaysUsageDescription` it's used to retrieve the location even when the app is backgrounded but we only need when the app is in use ([Apple ref](https://developer.apple.com/documentation/corelocation/choosing_the_authorization_level_for_location_services))

Reply:
Removed